pkg/external_repos/introspect.go

```diff
@@ -84,7 +86,13 @@ func Introspect(repo *dao.Repository, repoDao dao.RepositoryDao, rpm dao.RpmDao)
 		return 0, err
 	}
 
-	if repo.Revision != "" && repomd.Revision != "" && repomd.Revision == repo.Revision {
+	checksumStr := ""
+	if repomd.RepomdString != nil && *repomd.RepomdString != "" {
+		sum := sha256.Sum256([]byte(*repomd.RepomdString))
+		checksumStr = hex.EncodeToString(sum[:])
```

Comment:
Does this package from the standard library accomplish the same thing? https://pkg.go.dev/encoding/hex#EncodeToString

Also, just curious, why do we need to encode the checksum here?

Reply:
doh! thats what i meant to use.  IDE autocomplete for the lose. 

The checksum comes through as a byte array of numbers.  We could just store those.. but converting it to hex matches what you normally see when you compute a sha256 checksum